src/run-cache.unit.test.ts

```
@@ -148,4 +148,153 @@ describe('RunCache', () => {
+    const testKey = 'context-test-key';
+    const testValue = 'context-test-value';
+    const testTtl = 1000;
+
+    await RunCache.set({ 
+      key: testKey, 
+      value: testValue,
+      ttl: testTtl
+    });
+
+    expect(capturedContext).toBeDefined();
+    expect(capturedContext!.key).toBe(testKey);
```

Biome (1.9.4)

[error] Forbidden non-null assertion.

Unsafe fix: Replace with optional chain operator ?. This operator includes runtime checks, so it is safer than the compile-only non-null assertion operator


(lint/style/noNonNullAssertion)